app/src/main/java/org/simple/clinic/registration/phone/RegistrationPhoneScreen.kt

```diff
@@ -81,6 +81,7 @@ class RegistrationPhoneScreen(
     }
     context.injector<Injector>().inject(this)
 
+    isdCodeEditText.setText(resources.getString(R.string.registrationphone_country_code, country.isdCode))
```

Comment:
I have a doubt regarding this, aren't we sending pre-fill fields effect in `Init` won't that trigger the `preFillUserDetails`, why did it not work?

Reply:
The issue was that in the implementation, the `preFillUserDetails` method would be called only if a registration entry existed, not when it was being created for the first time. On the ISD code would get pre-filled on screen restoration, but not creation.

Since the code is part of the country and will not change with the model, I moved it out.